### ISSUE TITLE
5.0.5 - Users by default redirected to /portal with error message

### ISSUE THREAD

**Issue by donlent** [ironmansoftware/powershell-universal#3694]
### Version

5.0.5

### Severity

Quality-of-life

### Environment

msi

### Steps to Reproduce

- Define a Blank App with an URL, for example /TestApp
- Login with a normal user that has no Admin Rights
- This will happen:

![image](https://github.com/user-attachments/assets/7cf67955-43bb-42a7-b0c0-704624adf2eb)

- The user can then re-write the Link to /TestApp and it will work...

### Expected behavior

```console
This error should not happen. The provided URL has to be the one where the redirection will happen once successfuly authenticated
```

### Actual behavior

```console
Non-Admin Users get this error upon login.
```

**Comment by adamdriscoll** [comment]
Can you check the system log for the error you are seeing? I can't reproduce this at the moment. I have a local account user, with no role assigned and one app. I see the portal but it is blank.

**Comment by donlent** [comment]
Dear Adam
Unfortunately I can't find anything in the system log... This error just doesn't show up. So strange... :-( Do I need to increas the debug level by any chance?

So you're saying that you createad a blank app with a custom url, then visited the link and logged-in with the user and it works w/o problems? so no redirects?

Thanks a lot for your incredible support!

**Comment by adamdriscoll** [comment]
Ahhh. I see what you're saying. If I visit the app URL, it redirects to me the login page. Once I login, it takes me to the portal and not the app. So while I don't see the error, I'm also not landing back on the app.

**Comment by donlent** [comment]
Yes, you're probably correct! :-)) indeed, the issue is that this /portal seems sort of an "entry point" when you login.. and if you're not authorized it may show you the error that I'm experiencing. 
Can I still do something for you to allow you a better debugging or do you have everything for the fix?

**Comment by adamdriscoll** [comment]
I'm all good. This will be resolved in 5.0.6.
